src/stores/provider-constants-slice.ts

```diff
@@ -0,0 +1,38 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from 'src/stores';
+
+interface ProviderConstantsState {
+	defaultPhpVersion: string;
+	defaultWordPressVersion: string;
+	allowedPhpVersions: string[];
+}
+
+const initialState: ProviderConstantsState = {
+	// Set reasonable defaults until provider constants are loaded
+	defaultPhpVersion: '8.3',
+	defaultWordPressVersion: 'latest',
+	allowedPhpVersions: [ '8.4', '8.3', '8.2', '8.1', '8.0', '7.4' ],
+};
```

Comment:
I think this initial state could use more "empty" values, as they will be overwritten when the `providerConstantsChanged` is called.